packages/embark-proxy/src/index.ts

```diff
@@ -71,14 +71,24 @@ export default class ProxyManager {
     this.wsPort = port + 1;
     this.isWs = (/wss?/).test(this.embark.config.blockchainConfig.endpoint);
 
-    this.proxy = await new Proxy({events: this.events, plugins: this.plugins, logger: this.logger})
+    this.proxy = await new Proxy({ events: this.events, plugins: this.plugins, logger: this.logger })
       .serve(
         this.embark.config.blockchainConfig.endpoint,
         this.host,
         this.isWs ? this.wsPort : this.rpcPort,
         this.isWs,
         null,
       );
+
+    // set up HTTP proxy as well
+    await new Proxy({ events: this.events, plugins: this.plugins, logger: this.logger })
+      .serve(
+        this.embark.config.blockchainConfig.endpoint,
+        this.host,
+        !this.isWs ? this.wsPort : this.rpcPort,
+        !this.isWs,
+        null,
+      );
```

Comment:
This whole section is done a bit wrong.
It should be something like:
```
// HTTP
this.proxy = await new Proxy({ events: this.events, plugins: this.plugins, logger: this.logger })
      .serve(
        this.embark.config.blockchainConfig.endpoint,
        this.host,
        this.rpcPort,
        false,
        null,
      );
if (this.ws){
this.proxy = await new Proxy({ events: this.events, plugins: this.plugins, logger: this.logger })
      .serve(
        this.embark.config.blockchainConfig.endpoint,
        this.host,
        this.wsPort,
        true,
        null,
      );
}
```

Reply:
whoops! totally forgot to remove that after testing. I had no way of testing WebSockets, so I started an HTTP proxy and meant to remove it. Removed.

Reply:
I still see the double Proxy. Didn't you say you removed? Is it a Github bug?